### PR TITLE
miscellaneous fixups

### PR DESCRIPTION
**Pull request by RandyMcMillan**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
 <body>
   <button id="reload-btn" class="reload-btn fixing-top">reload page</button>
   <div class="centering">
-    <form mathod="post" action="sub" name="post-form" class="fixing-bottom">
+    <form method="post" action="sub" name="post-form" class="fixing-bottom">
```
Comment (RandyMcMillan) [inline templates/index.html:66]: feel free to cherry-pick this commit.

Comment (RandyMcMillan) [reply templates/index.html:66]: the other commits are (functionally) trivial.
this one needs to be merged.

**Comment by RandyMcMillan** [comment]
when editing - you should show invisible characters.
trailing white space is normally not desirable - except when it is syntactically necessary. :)

File: templates/index.html (hunk 1 above)
Comment (ryogrid) [reply templates/index.html:66]: @RandyMcMillan very very thanks to your contribution!

**Comment by ryogrid** [comment]
@RandyMcMillan 
very thank to your contribution!

**Pull request merged**